xbmc/ServiceBroker.cpp

```diff
@@ -57,6 +57,11 @@ void CServiceBroker::CreateLogging()
   g_serviceBroker.m_logging = std::make_unique<CLog>();
 }
 
+bool CServiceBroker::IsLoggingUp()
+{
+  return g_serviceBroker.m_logging && g_serviceBroker.m_logging->IsLogLevelLogged(LOG_LEVEL_DEBUG);
```

Comment:
id be careful here. The function name doesnt have anything to do with debugging, but the internals enforce that.

Id just leave it with the g_serviceBroker.m_logging check i think.

@lrusak i dont have time to check atm, but do we already have stuff in place to give us state for CLog? ie, is this needed, or is there some other pre-existing function that gives us the same CLog is up/down state?

Reply:
Thanks, I've changed it :+1: